docs/README.adoc

```diff
@@ -34,14 +34,44 @@ link:./integrating-libaktualizr.adoc[integrating-libaktualizr.adoc] - How to use
 
 Additional documentation intended for developers that may need to use the libaktualizr API should refer to link:https://advancedtelematic.github.io/aktualizr/index.html[the doxygen output on github].
 
+[NOTE]
+====
+The link above is for the doxygen docs on master. Doxygen docs for the following tagged versions are also available:
+
+* https://advancedtelematic.github.io/aktualizr/2018.8/index.html[2018.8]
+* https://advancedtelematic.github.io/aktualizr/2018.9/index.html[2018.9]
+* https://advancedtelematic.github.io/aktualizr/2018.11/index.html[2018.11]
+* https://advancedtelematic.github.io/aktualizr/2018.12/index.html[2018.12]
+* https://advancedtelematic.github.io/aktualizr/2018.13/index.html[2018.13]
+====
```

Comment:
Do we want to list these here, or would it be better if they were linked on the releases page somehow?

Reply:
Just added the links to the releases page.